opensearch-operator/pkg/helpers/helpers.go

```diff
@@ -112,7 +112,9 @@ func UsernameAndPassword(k8sClient k8s.K8sClient, cr *opsterv1.OpenSearchCluster
 		return string(username), string(password), nil
 	} else {
 		// Use default demo credentials
-		return "admin", "admin", nil
+		// minimum 8 character password and must contain at least one uppercase letter,
+		// one lowercase letter, one digit, and one special character
+		return "admin", "0penS3@rch!", nil
```

Comment:
I'm not sure about this part. `admin` didn't meet the default security requirements. Maybe this should be randomly generated instead (it'll get stored in the Secret object for admins to retrieve).

If this should be randomly generated, I could use some guidance if there is any existing code to assist or commonly used libraries.